### ISSUE TITLE
RP maintenance causes INTERNALERROR and no tests are run

### ISSUE THREAD

**Issue by pshv** [reportportal/agent-python-pytest#125]
We've just faced the issue that in case if ReportPortal is on the maintenance mode, it breaks whole test run and tests didn't even start due to INTERNALERROR. Pytest ver - 5.0.1

```
INTERNALERROR> Traceback (most recent call last):
INTERNALERROR>   File "c:\jenkins\shiningpanda\jobs\a4725e65\virtualenvs\f5ace85d\lib\site-packages\reportportal_client\service.py", line 67, in _get_json
INTERNALERROR>     return response.json()
INTERNALERROR>   File "c:\jenkins\shiningpanda\jobs\a4725e65\virtualenvs\f5ace85d\lib\site-packages\requests\models.py", line 897, in json
INTERNALERROR>     return complexjson.loads(self.text, **kwargs)
INTERNALERROR>   File "C:\Python37\Lib\json\__init__.py", line 348, in loads
INTERNALERROR>     return _default_decoder.decode(s)
INTERNALERROR>   File "C:\Python37\Lib\json\decoder.py", line 337, in decode
INTERNALERROR>     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
INTERNALERROR>   File "C:\Python37\Lib\json\decoder.py", line 355, in raw_decode
INTERNALERROR>     raise JSONDecodeError("Expecting value", s, err.value) from None
INTERNALERROR> json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
INTERNALERROR> 
INTERNALERROR> During handling of the above exception, another exception occurred:
INTERNALERROR> 
INTERNALERROR> Traceback (most recent call last):
INTERNALERROR>   File "c:\jenkins\shiningpanda\jobs\a4725e65\virtualenvs\f5ace85d\lib\site-packages\_pytest\main.py", line 211, in wrap_session
INTERNALERROR>     config.hook.pytest_sessionstart(session=session)
INTERNALERROR>   File "c:\jenkins\shiningpanda\jobs\a4725e65\virtualenvs\f5ace85d\lib\site-packages\pluggy\hooks.py", line 289, in __call__
INTERNALERROR>     return self._hookexec(self, self.get_hookimpls(), kwargs)
INTERNALERROR>   File "c:\jenkins\shiningpanda\jobs\a4725e65\virtualenvs\f5ace85d\lib\site-packages\pluggy\manager.py", line 87, in _hookexec
INTERNALERROR>     return self._inner_hookexec(hook, methods, kwargs)
INTERNALERROR>   File "c:\jenkins\shiningpanda\jobs\a4725e65\virtualenvs\f5ace85d\lib\site-packages\pluggy\manager.py", line 81, in <lambda>
INTERNALERROR>     firstresult=hook.spec.opts.get("firstresult") if hook.spec else False,
INTERNALERROR>   File "c:\jenkins\shiningpanda\jobs\a4725e65\virtualenvs\f5ace85d\lib\site-packages\pluggy\callers.py", line 208, in _multicall
INTERNALERROR>     return outcome.get_result()
INTERNALERROR>   File "c:\jenkins\shiningpanda\jobs\a4725e65\virtualenvs\f5ace85d\lib\site-packages\pluggy\callers.py", line 80, in get_result
INTERNALERROR>     raise ex[1].with_traceback(ex[2])
INTERNALERROR>   File "c:\jenkins\shiningpanda\jobs\a4725e65\virtualenvs\f5ace85d\lib\site-packages\pluggy\callers.py", line 187, in _multicall
INTERNALERROR>     res = hook_impl.function(*args)
INTERNALERROR>   File "c:\jenkins\shiningpanda\jobs\a4725e65\virtualenvs\f5ace85d\lib\site-packages\pytest_reportportal\plugin.py", line 58, in pytest_sessionstart
INTERNALERROR>     retries=int(session.config.getini('retries')),
INTERNALERROR>   File "c:\jenkins\shiningpanda\jobs\a4725e65\virtualenvs\f5ace85d\lib\site-packages\pytest_reportportal\service.py", line 100, in init_service
INTERNALERROR>     self.project_settiings = self.RP.rp_client.get_project_settings() if self.RP else None
INTERNALERROR>   File "c:\jenkins\shiningpanda\jobs\a4725e65\virtualenvs\f5ace85d\lib\site-packages\reportportal_client\service.py", line 237, in get_project_settings
INTERNALERROR>     return _get_json(r)
INTERNALERROR>   File "c:\jenkins\shiningpanda\jobs\a4725e65\virtualenvs\f5ace85d\lib\site-packages\reportportal_client\service.py", line 72, in _get_json
INTERNALERROR>     "Invalid response: {0}: {1}".format(value_error, response.text))
INTERNALERROR> reportportal_client.errors.ResponseError: Invalid response: Expecting value: line 1 column 1 (char 0): <!DOCTYPE html>
INTERNALERROR> <html lang="en" xmlns="http://www.w3.org/1999/xhtml">
INTERNALERROR>     <head>
INTERNALERROR>         <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
INTERNALERROR>         <meta charset="utf-8">
INTERNALERROR>         <title>Report Portal - Maintenance</title>
```

**Comment by prasanna1234** [comment]
Hi,

I am also receiving this same error. Any updates on this?

**Comment by NeuroDuck** [comment]
In pytest.ini, I set:
`rp_ignore_errors = True`
and then the test can proceed regardless of any connectivity problem to RP.

**Comment by iivanou** [comment]
Fixed in #212.
